### ISSUE TITLE
D5 – Baseline SVM classifier

### ISSUE THREAD

**Issue by DSkilton** [DSkilton/PictoGlyph#29]
**Description**
Implement Support Vector Machine baseline classifier.

**Tasks**
Use embeddings generated in D1.

**Train:**
sklearn.svm.SVC

**Evaluate:**
Accuracy
Precision
Recall
F1

**Acceptance Criteria**
Model trains successfully
Results recorded for comparison
